### ISSUE TITLE
Broken WMI filter

### ISSUE THREAD

**Issue by dbeugger** [ActiveDirectoryManagementFramework/ADMF#58]
```
PS>  Get-Module admf | ft Name, Version
Name Version
---- -------
ADMF 1.13.100
```

I created ADMF psd1 file for the WMI filter **Amdf-Test-1Q**
```
@{
    Name        = 'Admf-Test-1Q'
    Description = 'Test for  ADMF, one query'
    Query       = @('SELECT * FROM Win32_OperatingSystem WHERE ProductType = 1')
    # Namespace = 'root\CIMv2'
    Author      = 'Daniel Beugger'
    CreatedOn   = '2024-06-24'
}
```
When I create the filter with Invoke-AdmfDomain the first time, it shows no error. But a Test-AdmfDomain will show a difference. The execution of Invoke-AdmfDomain will show the following error
_WARNING: [20:03:18][Invoke-DMWmiFilter] Invalid input: 2! This command only accepts output objects from
Test-DMWmiFilter_

When I start gpmc.msc and edit the filter, no query is displayed. I can add the query by hand. Bit when i try to save the filter I get following error:
_Error text not available. Error code = 80041008_

I created the same WMI filter by hand with the same query. After SELECT I entered two blanks. Test-AdmfDomain shows a difference. Invoke-AdmfDomain changes the filter without an error. But again, the filter is corrupt

When I use a filter **Admf-Test-2Q** with two query's the creation works as expected
```
@{
    Name        = 'Admf-Test-2Q'
    Description = 'Test for ADMF, two querys'
    Query       = @(
        'SELECT * FROM Win32_OperatingSystem WHERE ProductType =  1'
        'SELECT * FROM Win32_OperatingSystem'
        )
    # Namespace = 'root\CIMv2'
    Author      = 'Daniel Beugger'
    CreatedOn   = '2024-06-24'
}
```
When the query's has changed the Invoke-AdmfDomain will fix it, but the same error as above will be shown:
_WARNING: [20:34:03][Invoke-DMWmiFilter] Invalid input: 2! This command only accepts output objects from
Test-DMWmiFilter_

**Comment by FriedrichWeinmann** [comment]
Thank you for your patience here and sorry for the wait :(
Found the issue here, waiting for fix on the other issue to release it. Very, very soon ... but not tonight ^^

**Comment by FriedrichWeinmann** [comment]
Fix has been released in 1.8.205

**Comment by dbeugger** [comment]
Did some successful testing with the new version 
Many thanks
